pkg/controllers/provisioning/scheduling/scheduling_benchmark_test.go

```diff
@@ -141,7 +141,7 @@ func TestSchedulingProfile(t *testing.T) {
 
 func benchmarkScheduler(b *testing.B, instanceCount, podCount int) {
 	// disable logging
-	ctx = logging.WithLogger(context.Background(), zap.NewNop().Sugar())
+	ctx = ctrl.IntoContext(context.Background(), logr.New(logr.Discard().GetSink()))
```

Comment:
Maybe just consider using the logger from `logging.go`? I don't think it's as big of a deal here since this is a test function, but probably just for cleanliness this is something that we should do